doc/man3/EVP_KDF_CTX_new_id.pod

```diff
@@ -0,0 +1,111 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF_CTX_new_id,
+EVP_KDF_CTX_free,
+EVP_KDF_CTX_ctrl,
+EVP_KDF_CTX_ctrl_str,
+EVP_KDF_CTX_ctrl_uint64,
+EVP_KDF_CTX_output_size,
+EVP_KDF_CTX_derive
+- EVP KDF algorithm functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/kdf.h>
+
+ EVP_KDF_CTX *EVP_KDF_CTX_new_id(int id);
+
+ void EVP_KDF_CTX_free(EVP_KDF_CTX *ctx);
+
+ int EVP_KDF_CTX_ctrl(EVP_KDF_CTX *ctx, int cmd, int p1, void *p2);
+
+ int EVP_KDF_CTX_ctrl_str(EVP_KDF_CTX *ctx, const char *type,
+                          const char *value);
+
+ int EVP_KDF_CTX_ctrl_uint64(EVP_KDF_CTX *ctx, int cmd, uint64_t value);
+
+ size_t EVP_KDF_CTX_output_size(EVP_KDF_CTX *ctx);
+
+ int EVP_KDF_CTX_derive(EVP_KDF_CTX *ctx, unsigned char *key, size_t *keylen);
```

Comment:
Another few nits:

- it would be nice if `EVP_KDF_CTX` was mentioned here, with the typedef from `ossl-typ.h` in the SYNOPSIS (we usually place it right after the `#include` line) and the name itself in the NAME section.  That way, people who want to start somewhere will be able to just `man EVP_KDF_CTX`.

- We usually don't have empty lines between every function in the SYNOPSIS, except possibly for grouping when there's a huge number of functions described (and even then, not always)...

Reply:
No worries - I am still learning what the conventions are.
Thanks for the guidance!